droughtpipeline/load.py

```diff
@@ -417,11 +406,71 @@ def send_to_ibf_api(
+                            exposure_pcodes = []
+                            for pcode in pcodes[f'{adm_level}']:
+                                amount = None
+                                forecast_admin = forecast_data.get_data_unit(
+                                    pcode, lead_time_event
+                                )
+                                if indicator == "population_affected":
+                                    amount = forecast_admin.pop_affected
+                                elif indicator == "population_affected_percentage":
+                                    amount = forecast_admin.pop_affected_perc
+                                elif indicator == "forecast_severity":
+                                    amount = forecast_admin.triggered # ( 1 if event_type == "trigger" else 0 )
+                                elif indicator == "forecast_trigger":
+                                    amount = forecast_trigger_status(
+                                        triggered=(forecast_admin.triggered >= 0),# True if event_type == "trigger" else False   ),
+                                        trigger_class=pipeline_will_trigger_portal,
+                                    )
+                                exposure_pcodes.append({
+                                    "placeCode": pcode, 
+                                    "amount": amount
+                                })
+                                processed_pcodes.append(pcode)
+
+                            body = {
+                                "countryCodeISO3": country,
+                                "leadTime": f"{lead_time_event}-month",
+                                "dynamicIndicator": indicator,
+                                "adminLevel": int(adm_level),
+                                "exposurePlaceCodes": exposure_pcodes,
+                                "disasterType": disasterType,
+                                "eventName": event_name,
+                                "date": upload_time,
+                            }
+                            statsPath=drought_extent.replace(".tif", f"_{event_name}_{lead_time_event}-month_{country}_{adm_level}.json" )
+                            statsPath=statsPath.replace("rainfall_forecast", f"{indicator}")
 
+                            with open(statsPath, 'w') as fp:
+                                json.dump(body, fp)
+                            
+                            self.ibf_api_post_request("admin-area-dynamic-data/exposure", body=body )
+                    processed_pcodes = list(set(processed_pcodes))
+                    
         # END OF EVENT LOOP
         ###############################################################################################################
 
-        # flood extent raster: admin-area-dynamic-data/raster/floods
+        # drought extent raster: admin-area-dynamic-data/raster/droughts
```

Comment:
maybe make a comment in this section (at the right place, not sure where exactly) explaining this data does update during the season, unlike exposure data.